src/main/java/com/softserveinc/ita/pageobjects/ProductPage.java

```diff
@@ -1,8 +1,62 @@
 package com.softserveinc.ita.pageobjects;
 
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.SelenideElement;
+import org.openqa.selenium.support.ui.Select;
+
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
 import static com.codeborne.selenide.Selenide.$x;
+import static java.time.Duration.ofSeconds;
+
+public class ProductPage extends BasePage<ProductPage> {
+
+    private final SelenideElement selectFromPriceModalMenuSelector = $("div > rz-sort > select");
+
+    public ProductPage setMinimalPrice(String price) {
```

Comment:
looks like, all methods are related to the search result page, isn't it?

Reply:
it still relevant, add above of class declaration: `//TODO: move methods not related to this page to other page objects`